torchgeo/datasets/utils.py

```diff
@@ -3,6 +3,9 @@
 
 """Common dataset utilities."""
 
+# https://github.com/sphinx-doc/sphinx/issues/11327
+from __future__ import annotations
```

Comment:
Had to use this hack to get our docs to build properly. This file gives a preview of what type hints will look like in Python 3.10 when `Optional` and `Union` become obsolete.

Reply:
Looks cool!